source/shader/LineShader.cpp

```diff
@@ -158,6 +165,36 @@ void LineShader::Init()
 
 
 void LineShader::Draw(const Point &from, const Point &to, float width, const Color &color, bool roundCap)
```

Comment:
Either remove roundCap or pass it to the gradient function.

Reply:
I forgor